CONTRIBUTING.md

```diff
@@ -183,3 +183,37 @@ Collect links at the end of each section, which are delimited by headings.
 
 [Here is a discussion of different line wrapping styles.]: https://web.archive.org/web/20220519121408/https://mtsknn.fi/blog/4-1-wrapping-styles-for-markdown-prose-and-code-comments/
 [reference links]: https://github.github.com/gfm/#reference-link
+
+### Terminology
+
+To avoid confusion around all the things called Nix, always use the following terms with capitalization as given.
+
+- Nix
+
+  Build system and package manager.
+
+  Read /nɪks/ ("Niks").
+
+- Nix language
+
+    Programming language to declare packages and configurations for Nix.
+
+  - Nix expression
+
+    Expression written in Nix language.
```

Comment:
@matta suggested that making Nix Language proper name by capitalization would avoid having to write "the Nix language" all the time.

```suggestion
- Nix Language

    Programming language to declare packages and configurations for Nix.

  - Nix expression

    Expression written in Nix Language.
```

Otherwise it would have to be

```suggestion
- Nix language

    Programming language to declare packages and configurations for Nix.

  - Nix expression

    Expression written in the Nix language.
```

Reply:
I was credited with the suggestion for "written in Nix Language" but please don't take that as indication I personally think it is a good idea.  :-)

I do think "written in Nix **L**anguage" (bold to emphasize capitalization) in this context is, possibly, _grammatically_ correct, this is true only if we are to assume that "Nix **L**anguage" is the proper name of the language and refer to it that way consistently. However, "Nix **L**anguage" is an odd and awkward name. I don't think this is a good idea for multiple reasons:

1. Very few, if any, languages of any sort, computer or otherwise, are named in English with multiple words, none that I know of include "language" in the proper name.
2. "written in Nix" or "written in the Nix language" are conventional, clear, with obvious meaning. There is no reason to use unconventional or unusual wording.